### ISSUE TITLE
Memcrashed vulnerability via default EE config

### ISSUE THREAD

**Issue by w33zy** [EasyEngine/easyengine#961]
Is this the default config of EE vulnerable to the Memcrashed DDOS attack that is currently crippling sites such as Gitlab, and Github?

https://blog.cloudflare.com/memcrashed-major-amplification-attacks-from-port-11211/

**Comment by VirtuBox** [comment]
Hello,

if there is a directive `-l 127.0.0.1` in your memcached configuration (/etc/memcached.conf), your server isn't vulnerable. 

However, it should not be a problem if you have configured properly your firewall

**Comment by w33zy** [comment]
@VirtuBox I thought that was the case as it seemed common sense, however the CF article I linked to mentioned that is you receive a response from `echo -en "\x00\x00\x00\x00\x00\x01\x00\x00stats\r\n" | nc -q1 -u 127.0.0.1 11211` your server is vulnerable.

**Comment by VirtuBox** [comment]
Yes, sorry I forgot to precise, you may have to add the line `-U 0` in /etc/memcached.conf.

But if you have configured a firewall on your server, it still not an issue. You can run the command 
```
nmap YOUR_SERVER_IP -p 11211 -sU -sS --script memcached-info
``` 
from another server to see if your memcached server is accessible from the outside.

**Comment by rahul286** [comment]
@w33zy Sorry for a delayed reply. I hope your issue is fixed by now with @VirtuBox answer.

EasyEngine uses redis for object-cache and full page cache in `--wpredis`. We did not have a single report of this troubling EE user.

If you are still facing the issue, please use http://community.rtcamp.com/c/easyengine for further support.

We are cleaning up issue tracker and closing this issue because we won't be able to contribute to this discussion further.
